### DISCUSSION TITLE
Multiple instances (again)?

### DISCUSSION THREAD

**Discussion by ghost** [mifi/lossless-cut#1387] [General]
Hello,

since quite a time losslesscut asks about what to do in the existing instance when one is calling the binary with a second video file.

Is there any way to open a second independent instance as before?  Would it be worth to make that possible?

**Comment by mifi** [comment]
Does the flag under https://github.com/mifi/lossless-cut/blob/master/cli.md help?

**Comment by ghost** [comment]
Mikael Finstad ***@***.***> writes:

> Does the flag under https://github.com/mifi/lossless-cut/blob/master/cli.md help?

Thanks for that hint.

It works half for me: the file argument at the end of the command line
is ignored, I have to open files from inside the program.  But then I
was able to edit two media files in two separate instances.

Here is the command line I had tried (with several different files in
order):

```
 /home/micha/software/LosslessCut-linux/losslesscut --no-sandbox --allow-multiple-instances "/home/micha/path/to/file"
```

**Reply by mifi** [reply]
Thanks for reporting. This was indeed a bug and I will fix it

**Comment by ghost** [comment]
Mikael Finstad ***@***.***> writes:

> Thanks for reporting. This was indeed a bug and I will fix it

Then: thanks in advance!
